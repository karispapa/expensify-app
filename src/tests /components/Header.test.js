import React from 'react';
import Header from '../../components/Header';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'

test('Should Render Header Correctly', ()=> {

  const wrapper  = shallow(<Header/>);
  expect(wrapper).toMatchSnapshot()

  // expect(wrapper.find('h1').text()).toBe('Expensify App')
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header/>)
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
})