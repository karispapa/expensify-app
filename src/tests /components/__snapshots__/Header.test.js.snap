// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Header Correctly 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
